scss/bitstyles/organisms/page-header/_index.scss

```diff
@@ -0,0 +1,66 @@
+@use '../../tools/media-query';
+@use '../../tools/classname';
+@use '../../tools/design-token';
+
+#{classname.get($classname-items: 'page-header', $layer: 'organism')} {
+  padding-top: var(design-token.get('size', 'm'));
+  background-color: var(design-token.get('color', 'grayscale', 'light-3'));
+}
+
+.o-page-header__content {
+  @include media-query.get('m') {
```

Comment:
Can we group all the media-query styles into one block at the end of the Sass (instead of several blocks, each inside their page-header__content-section class)? That way, the CSS that’s generated is clearer and smaller